kernel-ewasm/tests/utils/tester.ts

```diff
@@ -188,6 +189,44 @@ export class Tester {
         }
     }
 
+    // Delete a procedure. This assumes the current entry procedure for the
```

Comment:
Extraneous comment

Reply:
Fixed in 9666b8c8af56df62526686b692b2c87cdd966af5.